doc/api/next_api_changes/behavior/22691-JMK.rst

```diff
@@ -0,0 +1,6 @@
+QuadMesh mouseover defaults to False
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+New in 3.5, `.QuadMesh.get_cursor_data` allows display of data values
+under the cursor.  However, this can be very slow for large meshes, so
+by `.QuadMesh.set_mouseover` defaults to *False*.
```

Comment:
```suggestion
by ``.QuadMesh.set_mouseover`` defaults to *False*.
```
? As this doesn't exist in 3.5.

Reply:
Or actually the attribute directly?

Reply:
Since it doesn't exist in 3.5, this should be the attribute.